File: Python PowerPoint Examples/ShapesApiDemo/getSlideShapeTextRange.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SlideShapes.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "0"
paragraph_index = 0
portion_index = 0
result = api.get_slide_shape_text_range(fileName, slide_index, shape_index, paragraph_index=paragraph_index,portion_index=portion_index,password=password, storage=storage,folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/getSlideShapeTextRange.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "SlideShapes.pptx";
$password = null;
$folder = "input";
$storage = null;
$slideIndex = 0;
$shapeIndex = "0";
$paragraphIndex = 0;
$portionIndex = 0;
$result = $apiInstance->getSlideShapeTextRange($name,$slideIndex,$shapeIndex,$paragraphIndex,$portionIndex, $password, $folder, $storage);
print_r($result);